festflow/templates/festflow/index.html

```diff
@@ -1,43 +1,395 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title>ANK 2016 - The Knowledge Fest</title>
-
-  <!-- Include bootstrap -->
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-
-</head>
-<body>
-  <h1>Hello World!</h1>
-  {% if user.is_authenticated %}
-  	<!-- User is logged in -->
-    <div class="user_details">
-      <p>You are logged in as <b>{{ user.username }}</b>!</p>
-      <p>First Name: {{ user.first_name }}</p>
-      <p>Last Name: {{ user.last_name }}</p>
-      <p>Email: {{ user.email }}</p>
-  	</div>
-  	<a href="/logout">Logout</a>
-  {% else %}
-  	<!-- User is not logged in -->
-    <h2>{{ profiles_count }} users have already registered! Register NOW!</h2>
-  	{% for backend in backends.backends %}
-  	<a href="/login/{{backend}}/?next=/"><div class="login-button {{backend}}">{{backend}}</div></a><br>
-  	{% endfor %}
-  {% endif %}
-
-
-  {% for event in all_events %}
-    <div class="event_container">
-      <a href="{{ event.get_absolute_url }}"><h2>{{ event.name }}</h2></a>
-      {% if event.logo %}
-        <img src="{{ event.logo.url }}" width="200px">
-      {% endif %}
-      <p>{{ event.description }}</p>
-      <p>{{ event.venue }}</p>
-    </div>
-  {% endfor %}
-</body>
-</html>
+<html lang="en">
+<head> 
+	<meta charset="utf-8"> 
+	<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+	<meta name="description" content="ANK 2016 - The Knowledge Fest by National Institute Of Technology Durgapur Maths N Tech Club">
+	<meta name="keywords" content="MNTC,Ank 2K16" /> 
+	<meta name="author" content="Maths n Tech Club"> 
+	<title>ANK 2016! #LetTheFunBegin </title>	
+	
+	<!-- SOCIAL MEDIA META -->
+	<meta property="og:title" content="ANK 2016 | The Knowledge Fest"/>
+	<meta property="og:url" content="http://ank.mntc.in" />
+	<meta property="og:type" content="website" />
+	<meta property="og:description" content="MNTC NIT Durgapur presents ANK 2016 | The Knowledge Fest" />
+	
+	{% load staticfiles %}
+	<!-- STYLE SHEETS-->
+	<link href="/static/css/bootstrap.min.css" rel="stylesheet">
+	<link href="/static/css/prettyPhoto.css" rel="stylesheet">
+	<link href="/static/css/font-awesome.min.css" rel="stylesheet">
```

Comment:
@rahilsmd this is not the proper way of including static files

Follow this:
https://docs.djangoproject.com/en/1.10/howto/static-files/

Example
<link href="{% static "css/bootstrap.min.css" %}" rel="stylesheet">